trace/cloud-trace-demo-app/app/demo-service-b.yaml.template

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
```

Comment:
is this template needed if the service b yaml isn't included?

Reply:
This template is needed to generate service b yaml file since the IP host is going to be dynamic.